daemon/graphdriver/aufs/mount.go

```diff
@@ -5,16 +5,25 @@ package aufs // import "github.com/docker/docker/daemon/graphdriver/aufs"
 import (
 	"os/exec"
 	"syscall"
+	"time"
 
 	"github.com/docker/docker/pkg/mount"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
 )
 
+func debugEBusy(mnt string) {
```

Comment:
We took this out some time ago because it was causing major performance problems.

Reply:
I am well aware :) studied each and every aufs commit out there...

This is different from what was implemented earlier in commit 0e539fec331cb9dbc4ef784b55516570b11affe2.
* it's `fuser` not `lsof`
* it is specific to this particular mount point
* it actually shows who's holding the mount
* it takes *much* less time

With about 300 running containers (ie 5000 aufs mounts plus a few system ones):
```
 time lsof > /dev/null

real	0m10.964s
user	0m4.202s
sys	0m6.648s
```

```
root@kir-ubu-18:~/engine# time fuser -v -M -m /var/lib/docker/aufs/mnt/6f9203bfdcf175639c60dee45077aa249ecce5b99fb8f53fc14b250f163718a1
                     USER        PID ACCESS COMMAND
/var/lib/docker/aufs/mnt/6f9203bfdcf175639c60dee45077aa249ecce5b99fb8f53fc14b250f163718a1:
                     root     kernel mount /var/lib/docker/aufs/mnt/6f9203bfdcf175639c60dee45077aa249ecce5b99fb8f53fc14b250f163718a1
                     root      11621 .rce. sleep

real	0m0.097s
user	0m0.017s
sys	0m0.080s
```